### DISCUSSION TITLE
setup after download fails with {"message":"Invalid handler specified"}

### DISCUSSION THREAD

**Discussion by TMoersch** [wlcrs/huawei_solar#268] [Setting up the integration]
I decided today to make an complete new installation from scratch on a different flush device
I could manage anything to work until I try now to install huawei_solar.
The download works, but not the setup??

```
Der Konfigurationsfluss konnte nicht geladen werden: {"message":"Invalid handler specified"}
```
Thanks

**Comment by wlcrs** [comment]
cfr. #264
